renew_certificate.sh

```diff
@@ -60,8 +60,11 @@ export SSL_CERT_DIR=/dev/null
 mv letsencrypt/signed.crt.tmp letsencrypt/signed.crt
 echo "Downloading intermediate certificate..."
 
-INTERMEDIATE=$(openssl x509 -issuer -noout -in letsencrypt/signed.crt | awk -F= '{print $NF}')
-wget --no-verbose --secure-protocol=TLSv1_2 -O - https://letsencrypt.org/certs/lets-encrypt-$INTERMEDIATE.pem > letsencrypt/intermediate.pem
+# intermediate uri parsing from https://stackoverflow.com/a/68637388
+INTERMEDIATE=$(openssl x509 -noout -text -in letsencrypt/signed.crt | awk '/^[ \t]+CA Issuers[ \t]+-[ \t]+URI:/ { print gensub(/^.*URI:(.*)$/,"\\1","g",$0); }')
```

Comment:
I think you have entware installed, right?
Can you please check whether awk is available with the qnap busybox and whether this command works with it.

Reply:
Yes it is. I just looked up where awk is located and which version is running:

```
[/opt/qnap-letsencrypt] # which awk
/bin/awk
[/opt/qnap-letsencrypt] # awk --help
BusyBox v1.01 (2022.06.22-22:54+0000) multi-call binary

Usage: awk [OPTION]... [program-text] [FILE ...]

Options:
	-v var=val		assign value 'val' to variable 'var'
	-F sep		use 'sep' as field separator
	-f progname		read program source from file 'progname'
```

Reply:
Ok. Fingers crossed this is available for all qnap systems. :-)